ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/dmdownload/clientlibs/site/less/modal.less

```diff
@@ -0,0 +1,36 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright [2017]  Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* applies to all modals */
+.cmp-modal { 
```

Comment:
Does this apply to all modals? If so why is it under `/apps/.../components/modals/dmdownload/...`? .. feels like it should be pulled up to /apps/clientlibs/... somewhere?